### PR TITLE
Add defaultAction to docs

### PR DESCRIPTION
**Pull request by archilex**
This is an awesome built in feature that definitely needs to be documented. I stumbled upon it when I had to implement this exact feature, but it was a welcome suprise to see you've already implemented it! So many uses cases! I chose an onboarding type example, but change it to anything you like.  

- [X] Code style has been fixed by running the `composer cs` command.
- [X] Changes have been tested to not break existing functionality.
- [X] Documentation is up-to-date.

### CODE REVIEW

**Comment by ralphjsmit** [comment]
Didn't know this existed, super-cool! I really needed this in the past when opening a modal on a page and forcing the user to perform the action otherwise the page wasn't allowed to be accessed. Perfect stuff.

**Comment by archilex** [comment]
@ralphjsmit Our need was kind of similar to yours. We needed to show a warning on a custom page that continuing would do XYZ and we needed the user to confirm this was ok before proceeding. If they canceled they'd be redirected back. So many use cases for this feature.

**Comment by ralphjsmit** [comment]
Exactly! Super nice & very useful. I messed around with it earlier with e.g. calling `mountAction()` from the `mount()`, but that didn't work. Calling `mountAction()` from a booted method did work iirc, but that wasn't also ideal. So this is super useful indeed and surprising that it's not yet documented. But glad it gets documented at last 🫡

**Comment by danharrin** [comment]
Thanks

File: packages/panels/docs/04-pages.md [hunk 1]
```diff
@@ -54,6 +54,35 @@ protected function getHeaderActions(): array
...
+You can also pass an array of arguments to the default action using the `$defaultActionArguments` property:
+
+```php
+public $defaultActionArguments = ['step' => 2];
+```
+
+Alternatively, you can open an action modal when a page loads by specifying the `defaultAction` as a query string parameter to the page:
+
+```
+/admin/products/edit/932510?defaultAction=onboarding
```
Comment (saade) [inline packages/panels/docs/04-pages.md:83]:
```suggestion
/admin/products/edit/932510?action=onboarding
```

Hey @archilex, @danharrin i think this is wrong? The correct query param is `action`. `defaultAction` doesn't work as Povilas just pointed in his last video.

Comment (saade) [reply packages/panels/docs/04-pages.md:83]: I know for sure that `action` works because i'm using it in production. Not sure about `defaultAction`.

Comment (wychoong) [reply packages/panels/docs/04-pages.md:83]: https://github.com/filamentphp/filament/blob/e90998fe6aa4681569f09b6ebb01e5346d3a1644/packages/actions/src/Concerns/InteractsWithActions.php#L42-L49

Comment (danharrin) [reply packages/panels/docs/04-pages.md:83]: it was already fixed in the docs, I just haven't published them since

**Pull request merged**